codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/RuntimeTypes.kt

```diff
@@ -176,6 +176,13 @@ data class RuntimeType(val name: String?, val dependency: RustDependency?, val n
         val PartialEq = std.member("cmp::PartialEq")
         val StdError = RuntimeType("Error", dependency = null, namespace = "std::error")
         val String = RuntimeType("String", dependency = null, namespace = "std::string")
+        val DeriveBuilder = RuntimeType("Builder", dependency = CargoDependency.DeriveBuilder, namespace = "derive_builder")
+
+        fun RequestSpecModule(runtimeConfig: RuntimeConfig) =
+            RuntimeType("request_spec", CargoDependency.SmithyHttpServer(runtimeConfig), "${runtimeConfig.crateSrcPrefix}_http_server::routing")
+
+        fun Router(runtimeConfig: RuntimeConfig) =
+            RuntimeType("Router", CargoDependency.SmithyHttpServer(runtimeConfig), "${runtimeConfig.crateSrcPrefix}_http_server::routing")
```

Comment:
may have mentioned this a while ago, but you may at some point want your own companion object in `codegen-server` that contains server-specific runtime types